src/main/java/miniProject_eveningPractice/ManagerSystem.java

```diff
@@ -0,0 +1,105 @@
+package miniProject_eveningPractice;
+
+import java.util.*;
+import java.util.stream.Collectors;
+
+import static java.util.stream.Collectors.groupingBy;
+
+public class ManagerSystem {
+
+    private List<Employee> employeeList = new ArrayList<>();
+    private Map<Integer, Employee> employeeByIdMap = new HashMap<>();
+
+    public void addEmployee(Employee newEmployee) throws EmployeeIdAlreadyExistsException {
+        for (Employee employee : employeeList) {
+            if (employee.getId() == newEmployee.getId()) {
+                throw new EmployeeIdAlreadyExistsException("Employee With Such Id Already Exists");
+            }
+        }
+        employeeList.add(newEmployee);
+        employeeByIdMap.put(newEmployee.getId(), newEmployee);
+    }
+
+    public void removeEmployee(int id) {
+        employeeList = employeeList.stream().filter(employee -> employee.getId() != id).toList();
+        employeeByIdMap.remove(id);
```

Comment:
i think you can simplify this:
```suggestion
        if(employeeByIdMap.containsKey(id)
        employeeByIdMap.remove(id);
        else throw Exception();
```